app/views/peerage/peers/show.html.erb

```diff
@@ -18,7 +18,9 @@
     Phone: <%=@peer.phone%>
   </address>
   <strong>Elevated by:</strong> <%=@peer.elevated_by%><br>
-  <strong>Elevation Date:</strong> <%=@peer.elevation_date%>
+  <strong>Elevation Date:</strong> <%=@peer.elevation_date%><br>
+  <strong><%=@peer.class.dependent_name.capitalize%> of:</strong> <%=@peer.superiors.map{|x| link_to x.sca_name, peer_path(peerage, x.slug)}.join(', ').html_safe%><br>
```

Comment:
Peer instances should have a method that returns `self.class.dependent_name.capitalize` to avoid the repetitious and potentially smelly `@peer.class.dependent_name.capitalize` here.